erp/management/commands/start_scheduler.py

```diff
@@ -16,6 +17,9 @@ def handle(self, *args, **options):
         schedule.every(notify_changed_erps.HOURS_CHECK).hours.do(
             notify_changed_erps.job, verbose=True
         )
+        schedule.every(6).hours.do(
+            import_centres_vaccination.job, verbose=True, is_scheduler=True
+        )
```

Comment:
I realize this will trigger the job every 6 hours **since the scheduler started**. What we rather need would be to run it 4 times a day at fixed times: 0h, 6h, 12h and 18h.

Reply:
No problem